### ISSUE TITLE
Do you know an example of Grant setup in SvelteKit project?

### ISSUE THREAD

**Issue by stephane-klein** [simov/grant#274]
Hi,

I want to configure an OAuth 2 authentication connected to self-hosted GitLab instance for a [SvelteKit](https://kit.svelte.dev/) SSR project.

Here is my personnel [exploration issue](https://github.com/stephane-klein/poc-svelte-oauth-passport-gitlab/issues/1).

I try to use `Grant` library instead [node-oauth](https://github.com/ciaranj/node-oauth#oauth20) or [passport-oauth2](https://github.com/jaredhanson/passport-oauth2)

Question: do you know an example of Grant setup in SvelteKit project?

I intend to try to setup `Grand` in a [SvelteKit hooks](https://kit.svelte.dev/docs/hooks#handle), based on https://github.com/simov/grant/blob/master/lib/handler/vercel.js source code :thinking: 

Best regards,  
Stéphane

**Comment by simov** [comment]
Is the SvelteKit supposed to have a server running? Last time I checked it was a purely browser framework. If it have it's own server then you basically need a Grant handler for it so that you can leverage everything already available in Grant.

**Comment by stephane-klein** [comment]
> Is the SvelteKit supposed to have a server running?

@simov yes: Svelte**Kit**

**Comment by simov** [comment]
Yes, I glanced at your commits and it seems like it is going to work. Generally speaking though, the design goal in Grant has always been to provide a minimal interface that works out of the box for all of the supported HTTP frameworks and Cloud providers. I do see how creating the handler for Grant might be an extra effort in your case.
